modules/space/rotation/spicerotation.cpp

```diff
@@ -63,6 +66,32 @@ namespace {
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
+    constexpr openspace::properties::Property::PropertyInfo UseLightTravelTimeInfo = {
+        "UseLightTravelTime",
+        "Use Light Travel Time",
+        "If set to true, the rotation will be updated based on light travel "
+        "time to the observer.",
```

Comment:
This needs a lot more description as it is a pretty confusing topic to begin with